packages/effect/test/DateTime.test.ts

```diff
@@ -457,4 +457,266 @@ describe("DateTime", () => {
+          // This is better - it moves forward instead of backward
+          if (hours === 2 && minutes === 30) {
+            strictEqual(utcString, "2025-03-30T00:30:00.000Z") // Documents improved behavior
+          }
+        } else {
+          // ${desc} -> REJECTED (would be correct behavior)
+        }
+      })
+    })
+
+    describe("disambiguation strategies", () => {
+      it("should handle 'compatible' disambiguation (default behavior)", () => {
+        // Test DST fall-back (repeated time): 03:00 happens twice in Europe/Athens on Oct 26, 2025
+        // At UTC 01:00, Athens switches from GMT+3 to GMT+2, so 03:00 occurs twice
+        const timeZone = DateTime.zoneUnsafeMakeNamed("Europe/Athens")
+        const maybeDateTime = DateTime.makeZoned(
+          { year: 2025, month: 10, day: 26, hours: 3, minutes: 0, seconds: 0, millis: 0 },
+          { timeZone, adjustForTimeZone: true, disambiguation: "compatible" }
+        )
+
+        if (Option.isSome(maybeDateTime)) {
+          const utcString = DateTime.formatIso(DateTime.toUtc(maybeDateTime.value))
+          // 'compatible' should choose the earlier occurrence (first 03:00 in GMT+3)
+          strictEqual(utcString, "2025-10-26T00:00:00.000Z")
+        } else {
+          throw new Error("makeZoned should not return None for valid time")
+        }
+      })
+
+      it("should handle 'earlier' disambiguation", () => {
+        // Test DST fall-back: choose the earlier of two possible times
+        const timeZone = DateTime.zoneUnsafeMakeNamed("Europe/Athens")
+        const maybeDateTime = DateTime.makeZoned(
+          { year: 2025, month: 10, day: 26, hours: 3, minutes: 0, seconds: 0, millis: 0 },
+          { timeZone, adjustForTimeZone: true, disambiguation: "earlier" }
+        )
+
+        if (Option.isSome(maybeDateTime)) {
+          const utcString = DateTime.formatIso(DateTime.toUtc(maybeDateTime.value))
+          // Should choose first occurrence (GMT+3, UTC 00:00)
+          strictEqual(utcString, "2025-10-26T00:00:00.000Z")
+        } else {
+          throw new Error("makeZoned should not return None for valid time")
+        }
+      })
+
+      it("should handle 'later' disambiguation", () => {
+        // Test DST fall-back: choose the later of two possible times
+        const timeZone = DateTime.zoneUnsafeMakeNamed("Europe/Athens")
+        const maybeDateTime = DateTime.makeZoned(
+          { year: 2025, month: 10, day: 26, hours: 3, minutes: 0, seconds: 0, millis: 0 },
+          { timeZone, adjustForTimeZone: true, disambiguation: "later" }
+        )
+
+        if (Option.isSome(maybeDateTime)) {
+          const utcString = DateTime.formatIso(DateTime.toUtc(maybeDateTime.value))
+          // Should choose second occurrence (GMT+2, UTC 01:00)
+          strictEqual(utcString, "2025-10-26T01:00:00.000Z")
+        } else {
+          throw new Error("makeZoned should not return None for valid time")
+        }
+      })
```

Comment:
You can model this with Option.map and assertSome from our test utils: https://github.com/Effect-TS/effect/blob/main/packages/vitest/src/utils.ts#L197